gardening/demo/product_product.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="industry_fsm_sale.field_service_product" model="product.product">
+        <field name="project_id" ref="industry_fsm.fsm_project"/>
```

Comment:
Really needed?!

Reply:
It permits to create a field service task for a repair